.github/workflows/build-llvm.yml

```diff
@@ -36,9 +51,18 @@ jobs:
           cd llvm-install
           tar -zcf ../llvm.tgz .
 
-      - uses: actions/upload-artifact@v4
+      - name: Upload LLVM for CPU
+        if: matrix.build-type == 'CPU'
+        uses: actions/upload-artifact@v4
         with:
           name: llvm-${{ env.LLVM_HASH }}
-          path: llvm.tgz
+          path: ./llvm-project/llvm.tgz
+
+      - name: Upload LLVM for IMEX
+        if: matrix.build-type == 'IMEX'
+        uses: actions/upload-artifact@v4
+        with:
+          name: llvm-${{ env.LLVM_HASH }}-imex-patched
+          path: ./llvm-project/llvm.tgz
```

Comment:
Is this enough to avoid artifact overlapping?

Reply:
I believe it should be enough. The artifacts will be stored under different names for imex and cpu builds, so they shouldn't overlap. @leshikus correct me if I'm wrong

Reply:
@dchigarev correct